### ISSUE TITLE
Showing full panorama in small window

### ISSUE THREAD

**Issue by mhandze** [mpetroff/pannellum#348]
Hello,

I have a question if it's possible and if yes then how, to show big panorama in 10x smaller preview.

I'm working with 2000px x 1000px panos and target preview would size is 200px x 100px. I'd like to show full 360 view of this pano in that small area. 

Here is example list of preview in right column:
http://imgur.com/vXupbZ8

**Comment by mpetroff** [comment]
If you want to show the full image, you don't need a viewer&mdash;you can just show the equirectangular image. Pannellum uses a rectilinear projection, which is mathematically unable to show 180 degrees or more.

**Comment by mhandze** [comment]
Hello, Thank you for reply. Yes, I could just show simple image, but the point is that I'm using that right list of panos in create hotspots process. On main big pano user is adding "source" for hotspot and on pano listed in right column adding "target" for hotspot.
If I will display simple image insted of panorama, there is no way for pannellum api to find "hotpot" params once it clicked, that's why I'm looking for way to show 360 as panorama..

**Comment by mpetroff** [comment]
I don't follow what you're saying.

**Comment by mhandze** [comment]
Sorry, so my question is if there is any way to create hospot on panorama or image showing full pano (360) like on screen from the link ?

**Comment by mpetroff** [comment]
As Pannellum only supports displaying rectilinear projected panoramas, this isn't supported. I'm sure you could build something like that using the API, though.

**Comment by mhandze** [comment]
Yes, I did it using 3 120 degrees panos displayed one after each other.

**Comment by Hfawad** [comment]
@mhandze  I want to do same thing you have stated in above comments. Can you help me in doing this?

**Comment by mhandze** [comment]
@Hfawad : I started with 3 x 120 degrees panos side by side, but from API's point of view they were 3 different panos, so keeping then as one logic for hotspots would need some extra workaround code. I ended up with one 120 degrees pano, which works fine so far.

**Comment by IHaveNoIdeaWhatIAmDoingHere** [comment]
@mhandze I like what you have there as a tour-creation tool (at leas what is to be seen in the picture you provided) Is that tool operational? Do you provide it?
Would love to hear more from you!

**Comment by mhandze** [comment]
@IHaveNoIdeaWhatIAmDoingHere the image was only on of the ideas for the interface. Whole thing is now still under development.

**Comment by kanalasumant** [comment]
@mhandze I really wanted to have the feature you're talking about, i.e having all the functionality of pannellum hotspots mapping, etc.. but with a regular image without any drag to scroll effect. Could you kindly assist me in this? Thanks!
